internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -538,7 +538,7 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 	return c, nil
 }
 
-func (h *Host) modifyHostSettings(ctx context.Context, containerID string, req *guestrequest.ModificationRequest) (err error) {
+func (h *Host) modifyHostSettings(ctx context.Context, containerID string, req *guestrequest.ModificationRequest) (retErr error) {
```

Comment:
I'm missing where `retErr` is ever written to?

Reply:
Oh, interesting.  The `retErr` is assigned to as part of the return.  For anyone else confused like I was:
https://go.dev/play/p/IDH-93ODPLs